### PR TITLE
Svls 4910 make sure next on crash

### PR DESCRIPTION
**Pull request by alexgallotta**
- remove all unwraps from non test code
- remove all warnings from clippy
- if active loops fail, run the noop loop to call /next

### CODE REVIEW

File: bottlecap/src/bin/bottlecap/main.rs [hunk 1]
```diff
@@ -168,7 +167,18 @@ async fn main() -> Result<()> {
         .map_err(|e| Error::new(std::io::ErrorKind::InvalidData, e.to_string()))?;
 
     if let Some(resolved_api_key) = resolve_secrets(Arc::clone(&config), &aws_config).await {
-        extension_loop_active(&aws_config, &config, &client, &r, resolved_api_key).await
+        match extension_loop_active(&aws_config, &config, &client, &r, resolved_api_key).await {
+            Ok(()) => {
+                info!("Extension loop completed successfully");
```
Comment (duncanista) [inline bottlecap/src/bin/bottlecap/main.rs:172]:
```suggestion
                debug!("Extension loop completed successfully");
```

**Review by astuyve — APPROVED** [review]

File: bottlecap/src/bin/bottlecap/main.rs (hunk 1 above)
Comment (astuyve) [reply bottlecap/src/bin/bottlecap/main.rs:172]: Yeah this should be a debug

**Review by duncanista — APPROVED** [review]
LGTM – make sure to update the `info` log to `debug` on the comment I left!

File: bottlecap/src/bin/bottlecap/main.rs (hunk 1 above)
Comment (alexgallotta) [reply bottlecap/src/bin/bottlecap/main.rs]: oh true! Good catch, thanks!

**Pull request merged**